protocol/wavefront/wavefrontlistener.go

```diff
@@ -139,7 +139,7 @@ func (listener *Listener) fromWavefrontDatapoint(line string) *datapoint.Datapoi
 	for _, p := range pieces {
 		dimPieces := strings.SplitN(p, "=", 2)
 		if len(dimPieces) == 2 {
-			dimensions[dimPieces[0]] = dimPieces[1]
+			dimensions[dimPieces[0]] = strings.Replace(dimPieces[1],"\"","",-1)
```

Comment:
don't put two completely different things in one pull request. 

also, this is too trivial to be functional.  i'll propose a thorough solution.